.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - '2.7'
+script:
+  - docker-compose run web /bin/bash -c "coverage run --source='.' ./manage.py test && coverage report"
```

Comment:
This is great that we can share the docker configuration with the test server, we never got that set up with LORE